### DISCUSSION TITLE
Request for Assistance with Laravel Spark Subscription Pending Issue

### DISCUSSION THREAD

**Discussion by cntabana** [filamentphp/filament#9761] [General]
Hello everyone,

I hope this message finds you well. I recently purchased Laravel Spark last week and successfully installed it. However, I am currently encountering an issue where, after completing the payment process, the system seems to get stuck on the "Subscription Pending" page. I'm not sure if this is related to Filament.

Upon further investigation, I've observed that the webhooks associated with the subscription process are being executed successfully, and the subscription data is indeed being stored in my database (both in the 'subscriptions' and 'subscription_items' tables). I am utilising FilamentPHP in my setup.

I have attached screenshots related to the issue for your reference.

<img width="1018" alt="Screenshot 2023-11-20 at 09 13 47" src="https://github.com/filamentphp/filament/assets/6320126/6538116e-b660-4145-86a5-1d51256ea27d">

<img width="1254" alt="Screenshot 2023-11-20 at 09 14 51" src="https://github.com/filamentphp/filament/assets/6320126/c5015569-6497-45f0-8960-0393451e8803">

My SparkServiceProvider:

 ```
public function boot(): void
    {
        Cashier::useCustomerModel(Organisation::class);

        Spark::billable(Organisation::class)->resolve(function (Request $request) {
            return $request->user()->getDefaultTenant(Filament::getPanel('admin'));
        });

        Spark::billable(Organisation::class)->authorize(function (Organisation $billable, Request $request) {
            return $request->user() &&
                   $request->user()->latest_organisation_id == $billable->id;
        });

        Spark::billable(Organisation::class)->checkPlanEligibility(function (Organisation $billable, Plan $plan) {
            // if ($billable->projects > 5 && $plan->name == 'Basic') {
            //     throw ValidationException::withMessages([
            //         'plan' => 'You have too many projects for the selected plan.'
            //     ]);
            // }
        });
    }
```

Although I reached out to Laravel for assistance, I'm still awaiting their response. If anyone has experienced a similar issue or has insights into potential solutions, I would greatly appreciate your help. I apologize if this isn't the right channel for such inquiries.

Thank you all for your time and assistance!

**Comment by arbet** [comment]
Facing the same issue here. Did you manage to solve this?
